services/worker/tests/fixtures/descriptive_statistics_dataset.py

```diff
@@ -1640,9 +1640,9 @@ def nan_column() -> list[None]:
     },
     features=Features(
         {
-            "audio": Audio(sampling_rate=1600),
-            "audio_nan": Audio(sampling_rate=1600),
-            "audio_all_nan": Audio(sampling_rate=1600),
+            "audio": Audio(sampling_rate=1600, decode=False),
+            "audio_nan": Audio(sampling_rate=1600, decode=False),
+            "audio_all_nan": Audio(sampling_rate=1600, decode=False),
```

Comment:
why this change ?

Reply:
to make `path` accessible here https://github.com/huggingface/dataset-viewer/pull/2793/files#diff-6cac2dec70bb036137022bcfb8bbe859ecc3d25b990558ef7efcb27bd75f5466R806